src/components/VmDetails/cards/NicsCard/NicListItem.js

```diff
@@ -41,18 +41,29 @@ const NicListItem = ({ idPrefix, nic, vmStatus, vnicProfileList, isEditing, onEd
       </div>
       <Grid>
         <Row>
-          <Col cols={4} className={style['ip4-container']} id={`${idPrefix}-ipv4`}>
+          <Col cols={4} wrapExpand className={style['ip4-container']} id={`${idPrefix}-ipv4`}>
             { nic.ipv4.length > 0 &&
-              <div>
-                {nic.ipv4.map(ip4 => <div key={`${nic.id}-${ip4}`}>{msg.nicIP4()}: {ip4}</div>)}
-              </div>
+              nic.ipv4.map((ip4, index) =>
+                <FieldValue
+                  tooltip={ip4}
+                  key={`${nic.id}-${ip4}-${index}`}
+                  id={`${idPrefix}-ipv4-${index}`}
+                >
+                  {msg.nicIP4()}: {ip4}
+                </FieldValue>)
             }
           </Col>
-          <Col cols={8} className={style['ip6-container']} id={`${idPrefix}-ipv6`}>
+          <Col cols={8} wrapExpand className={style['ip6-container']} id={`${idPrefix}-ipv6`}>
             { nic.ipv6.length > 0 &&
-              <div>
-                {nic.ipv6.map(ip6 => <div key={`${nic.id}-${ip6}`}>{msg.nicIP6()}: {ip6}</div>)}
-              </div>
+              nic.ipv6.map((ip6, index) =>
+                <FieldValue
+                  className={style['ip']}
```

Comment:
The `className` on the ip6 `FieldValue` needs to be consistent with the one on the ip4 `FieldValue`.